packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -210,22 +210,21 @@ interface UserMenuProps {
 ```
  * @param props.account Object that contains information about the active user.
 *   {firstName: 'string', lastName: 'string', email: 'string'}
- * @param props.activeProduct MongoDB product that is currently active.
- *  Possible values: ['account', 'charts', 'cloud', 'realm', 'support', 'university']
  * @param props.activeNav Determines what nav item is currently active.
  * @param props.onLogout Callback fired when a user logs out.
  * @param props.onProductChange Callback invoked after the user clicks a product.
  * @param props.hosts Object where keys are MDB products and values are the desired hostURL override for that product, to enable `<UserMenu />` to work across all environments.
  * @param props.urls Object to enable custom overrides for every `href` used in `<UserMenu />`.
+ * @param props.activePlatform MongoDB platform that is currently active.
  */
 function UserMenu({
   account,
-  activeProduct,
   activeNav,
   onLogout: onLogoutProp,
   onProductChange = () => {},
   urls: urlsProp,
   hosts: hostsProp,
+  activePlatform,
```

Comment:
Hmm... A question for both @brookescarlett and @hswolff: Should we use `activeProduct` to create a smart default value for this? Given that the current implementation is only causing an issue in one edge case, it indicates to me that this prop may only need to exist for those cases where `activeProduct` isn't cutting it.

Reply:
We we're doing this before, can point you to thread on slack for how this came about

Reply:
We're defaulting it to Cloud in MongoNav because cloud applications (Realm, Charts, Atlas) are the only ones consuming the entire nav right now and therefore this will be accurately passed down to UserMenu

Reply:
> We we're doing this before, can point you to thread on slack for how this came about

This, basically. `activeProduct` was being overloaded in how it was being used hence the need for this new distinction.